TYPO3.Flow/Classes/TYPO3/Flow/Core/Bootstrap.php

```diff
@@ -590,6 +602,12 @@ protected function ensureRequiredEnvironment()
                 exit(1);
             }
         }
+        if (!is_dir(FLOW_PATH_TEMPORARY) && !is_link(FLOW_PATH_TEMPORARY)) {
+            if (!@mkdir(FLOW_PATH_TEMPORARY, 0777, true)) {
```

Comment:
I just copied the code from `Files::createDirectoryRecursively()`. We can't use that method from here because the exception handler is not registered at this point